tests/functional/test_manu_thermoporomech_nofrac.py

```diff
@@ -253,6 +254,7 @@ def actual_ooc(material_constants: dict) -> list[list[dict[str, float]]]:
                     "meshing_arguments": {"cell_size": 0.25},
                     "perturbation": 0.3,
                     "heterogeneity": 10.0,
+                    "times_to_export": [],  # Suppress output for tests
```

Comment:
Have you somehow tried to check whether you have caught all instances where this is needed? I.e., are any vtu files are written by the verification tests that were not written previously?

Reply:
I checked the contents of `tests/functional` and file IO is purged from there.
Started doing so in `tests/models` but some of the set-ups are quite nested, I did not manage to get every one.

Reply:
Forgive me if this is a stupid question: where does that leave us?

Reply:
I guess I got that onto myself.
I was able to purge it everywhere after some search. It now happens only in `tests/viz` where it is explicitly tested.